src/chainedvector.jl

```diff
@@ -546,8 +546,8 @@ function Base.resize!(A::ChainedVector{T, AT}, len) where {T, AT}
         resize!(A.arrays, chunk)
         resize!(A.inds, chunk)
         # resize individual chunk
-        resize!(A.arrays[chunk], A.inds[chunk] - len)
-        A.inds[chunk] -= A.inds[chunk] - len
+        resize!(A.arrays[chunk], length(A.arrays[chunk]) - (A.inds[chunk] - len))
```

Comment:
An alternative (unsure what is more readable):
```suggestion
        resize!(A.arrays[chunk], chunk == 1 ? len : len - A.inds[chunk - 1])
```

Reply:
I prefer branchless code and `length` should be O(1) in most cases... But its your call.